### PR TITLE
improvement(client-debugger-view): Merge Container Summary view with summary header

### PR DESCRIPTION
**Pull request by Josmithr**
The existing debugger view contained a static header with a subset of a Container Summary, and there was a separate tab view that included a more complete view of the same data. This PR updates the static header to include all of the relevant data, and removes the dedicated tab.

Also...
* removes the `resolvedUrl` field, which is not a part of our spec.
* cleans up iconInitialization to remove console errors and ensure icons are loaded in the right places.
* adds a few of TODO comments

### CODE REVIEW

File: packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+let iconsInitialized = false;
+
+/**
+ * Initialize Fluent icons used this library's components, if they have not already been initialized.
+ */
+export function initializeFluentUiIcons(): void {
+	if (!iconsInitialized) {
+		initializeIcons();
+		iconsInitialized = true;
```
Comment (chentong7) [inline packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts:16]: I think we can return the boolean value here to check whether icon has been initialized.

Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts:16]: By the time this function returns, that will always be true. Is there a particular use-case you're thinking of?

File: packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { initializeIcons } from "@fluentui/react";
+
+let iconsInitialized = false;
+
+/**
+ * Initialize Fluent icons used this library's components, if they have not already been initialized.
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts:11]:
```suggestion
 * Initialize Fluent icons used this library's components, if they have not already been initialized.
 *
 * @remarks Wraps FluentUI's icon initialization, ensuring it is only called once.
 * FluentUI reports errors if this is called more than once, but as this library exports multiple entry-points, it is impossible to say which module needs to call this.
```

File: packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * @remarks Wraps FluentUI's icon initialization, ensuring it is only called once.
+ * FluentUI reports errors if this is called more than once, but as this library exports multiple entry-points, it is impossible to say which module needs to call this.
```
Comment (Josmithr) [inline packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts:14]:
```suggestion
 * FluentUI reports errors if this is called more than once, but as this library exports multiple entry-points,
 * it is impossible to say which module needs to call this.
```

File: packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts (hunk 1 above)
Comment (chentong7) [reply packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts:20]: we just want to make sure the icon only initialized once all the time, right? Is that other place call initializeFluentUiIcons()?

**Review by chentong7 — APPROVED** [review]
LGTM

File: packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts (hunk 1 above)
Comment (Josmithr) [reply packages/tools/client-debugger/client-debugger-view/src/InitializeIcons.ts:20]: Right. So yeah, any module containing components that use FluentUI icons should call this.

**Pull request merged**